tslint.json

```diff
@@ -1,6 +1,31 @@
 {
   "extends": ["tslint:latest", "tslint-sonarts", "codelyzer", "tslint-config-prettier", "./packages/stark-build/config/tslint.json"],
   "rules": {
-    "component-selector": false
+    "component-selector": false,
+    "completed-docs": [
+      true,
+      {
+        "enums": true,
+        "variables": {
+          "tags": { "existence": ["ignore", "link", "param", "returns"] }
+        },
+        "functions": {
+          "tags": { "existence": ["ignore", "link", "param", "returns"] }
+        },
+        "interfaces": {
+          "tags": { "existence": ["ignore", "link", "param", "returns"] },
+          "visibilities": ["exported", "internal"]
+        },
+        "classes": {
+          "tags": { "existence": ["ignore", "link", "param", "returns"] },
+          "visibilities": ["internal"]
+        },
+        "methods": {
+          "tags": { "existence": ["ignore", "link", "param", "returns"] }
+        }
+      }
+    ],
+    "jsdoc-format": [true, "check-multiline-start"],
+    "no-redundant-jsdoc": true
```

Comment:
I think these 3 new rules should be also added to these packages:
- stark-core
- stark-ui
- starter

This is becasue the "npm run lint:all" script that is executed in Travis will lint all those packages separately ;)